### DISCUSSION TITLE
Possible to assign different keys BUT only 1 knob to adjustment sliders in Lightroom?

### DISCUSSION THREAD

**Discussion by aomatani** [rsjaffe/MIDI2LR#785] [Q&A] [answered]
Hi all,
I'm new to MIDI2LR. In Lightroom, I am wondering if you can assign keys to "activate" (bring focus to) different sliders, and assign knobs for adjusting them? For example, assign "F16" to "activate" the White Balance slider, and assign a knob for the actual adjustment.

The reason being that I only have a small controller, and there are not enough knobs to assign to every slider, so it'll be great if I can use different keys for different sliders, and just use a single knob to adjust each and every one of them.

Thank you for any insights or workarounds!

**Comment by rsjaffe** [comment, marked as answer]
Yes. See https://github.com/rsjaffe/MIDI2LR/wiki/Commands#9 More--Less. If you can move focus to a specific slider, you can then use that on your knob.

**Reply by aomatani** [reply]
Thank you for your reply @rsjaffe

In that case, is it possible to assign keys to jump/move focus to specific sliders? For eg F7 for Temperature, F8 for Tint etc. 
I have only one main knob, so this means that this one and only knob is able to make adjustments to different sliders based on which key is pressed, i.e. it adjusts for Temperature if F7 is pressed, and adjusts for Tint when F8 is pressed.
 
Sorry if I misunderstood you🙏🏻

**Reply by Surfingserenity** [reply]
That's a good way to manage only having limited knobs, it's even worth setting up even if you have many knobs as it's convenient to just keep you hand on one knob and switch sliders using buttons

**Reply by themeadery** [reply]
I've been looking for this functionality for a while. Thank you so much for bringing it to my attention and linking right to the documentation!

**Comment by mediahope** [comment]
If I understood the question correctly, then yes - it is possible to do this by switching profiles in MIDI2LR.

**Reply by Surfingserenity** [reply]
How would that let you jump to different sliders?
